src/Renderer.js

```diff
@@ -55,6 +55,73 @@ export default class Renderer {
     return nodeEl;
   }
 
+  // extract all the literals, create clones, and absolutely position 
+  // them at their original locations
+  animateTransition(ast, toBlocks) {
+    var that = this, rootNodes = ast.rootNodes;
```

Comment:
ditch `var that = this` pattern. It's unnecessary in ES6.

When you want a function to have the same `this` as it's parent scope, you can just use the arrow syntax. i.e.

```
this.a = 'foo';
var printA = () => { console.log(this.a); }
printA()
```
will print 'foo'